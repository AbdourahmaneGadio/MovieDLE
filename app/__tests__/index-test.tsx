import {
  cleanup,
  render,
} from '@testing-library/react-native';

import Index from '@/app/index';

afterEach(cleanup);

describe('<Index />', () => {
  test('Index renders correctly', () => {
    const tree = render(<Index />).toJSON;
    expect(tree).toMatchSnapshot();
  });
});
