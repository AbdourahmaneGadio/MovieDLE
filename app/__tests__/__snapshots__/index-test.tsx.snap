// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Index /> Index renders correctly 1`] = `[Function]`;
